examples/grpc-server-with-prometheus/client/client.go

```diff
@@ -3,32 +3,83 @@ package main
 import (
 	"fmt"
 	"log"
+	"net/http"
 
 	"golang.org/x/net/context"
 	"google.golang.org/grpc"
+	"github.com/grpc-ecosystem/go-grpc-prometheus"
 
 	pb "github.com/grpc-ecosystem/go-grpc-prometheus/examples/grpc-server-with-prometheus/protobuf"
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/client_golang/prometheus/promhttp"
+	"os"
```

Comment:
Please standardize the order of packages in report field. 
https://github.com/golang/go/wiki/CodeReviewComments#imports